### PR TITLE
Ensure all parachain configuration extrinsics are operational

### PR DESCRIPTION
**Pull request by KiChjang**
Re-add `DispatchClass::Operational` to all extrinsic weights in `parachain_runtime::configuration`. They were erroneously removed when benchmarks for the configuration pallet was introduced in #3862.

### CODE REVIEW

**Review by shawntabrizi — APPROVED** [review]

File: runtime/parachains/src/configuration.rs [hunk 1]
```diff
@@ -670,15 +757,21 @@ pub mod pallet {
 		}
 
 		/// Sets the number of sessions after which an HRMP open channel request expires.
-		#[pallet::weight(T::WeightInfo::set_hrmp_open_request_ttl())]
+		#[pallet::weight((
+			T::WeightInfo::set_hrmp_open_request_ttl(),
+			DispatchClass::Operational,
+		))]
```
Comment (emostov) [inline runtime/parachains/src/configuration.rs:763]: Seems irrelevant to add it here, but also don't see the harm

**Review by emostov — APPROVED** [review]
I don't have much context on this module, but looks reasonable considering they all require root origin.

**Comment by KiChjang** [comment]
bot merge
